sound/soc/sof/topology.c

```diff
@@ -394,6 +394,7 @@ static const struct sof_process_types sof_process[] = {
 	{"KEYWORD_DETECT", SOF_PROCESS_KEYWORD_DETECT, SOF_COMP_KEYWORD_DETECT},
 	{"KPB", SOF_PROCESS_KPB, SOF_COMP_KPB},
 	{"CHAN_SELECTOR", SOF_PROCESS_CHAN_SELECTOR, SOF_COMP_SELECTOR},
+	{"MUX", SOF_PROCESS_MUX, SOF_COMP_MUX},
```

Comment:
should there be a DEMUX as well?

Reply:
@plbossart, yes there should be, this version was done with @akloniex first mux version . I'll update once we are aligned with driver and fw. We are again out of sync... see 
https://github.com/thesofproject/sof/pull/1304 and https://github.com/thesofproject/linux/pull/864